local/compose/logs.go

```diff
@@ -48,6 +48,7 @@ func (s *composeService) Logs(ctx context.Context, projectName string, consumer
 				ShowStdout: true,
 				ShowStderr: true,
 				Follow:     options.Follow,
+				Since:      options.Since,
```

Comment:
docker API `Since` is documented as a Unix Timestamp (https://docs.docker.com/engine/api/v1.41/#operation/ContainerLogs)
but according to [docker/ci code](https://github.com/docker/cli/blob/master/cli/command/container/logs.go#L61) just passing to the engine seems to be ok.
Confirmed by actual engine code https://github.com/moby/moby/blob/7b9275c0da707b030e62c96b679a976f31f929d3/api/types/time/timestamp.go#L20-L26
That's a bit weird, but ¯\\_(ツ)_/¯